clang/tools/driver/CacheLauncherMode.cpp

```diff
@@ -125,6 +125,20 @@ static void addCommonArgs(bool ForDriver, SmallVectorImpl<const char *> &Args,
 /// Arguments specific to \p clang-cache compiler launcher functionality.
 static void addLauncherArgs(SmallVectorImpl<const char *> &Args,
                             llvm::StringSaver &Saver) {
+  auto RemotePath =
+      llvm::sys::Process::GetEnv("LLVM_CACHE_REMOTE_SERVICE_SOCKET_PATH");
+
+  if (!llvm::sys::Process::GetEnv("CLANG_CACHE_DISABLE_MCCAS") && !RemotePath) {
```

Comment:
It doesn't matter functionally, but aesthetically I'd prefer to add these options after the other existing options that get added in this function, not having them the first thing at the top of the function.